r/examples/example_rsparkling.R

```diff
@@ -15,13 +15,17 @@
 # devtools::install_github("h2oai/sparkling-water", subdir = "/r/rsparkling")
 # devtools::install_github("rstudio/sparklyr")
 
+#Set version of spark. 
+#Note: At the time of spark_connect sparklyr will call the spark_dependencies function in the rsparkling package.
+options(rsparkling.sparklingwater.version = "1.6.7")
```

Comment:
What i do not like on this one is that it forces explicit order of operations: you need to setup version first and then call  `spark_connect`. There was already a problem report in community of forum which was reporting error in RSparkling, because of wrong operations order. This even more stress user. 

But I understand that `spark_connect(master = ..., extensions = add(rsparkling(version="1.6.8")) )` is not best one as well.